analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2021 HERE Europe B.V.
+ * Copyright (C) 2017-2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
The new Copyright statement should just have the inception year.

Reply:
This is fixed after rebasing on your copyright cleanup PR.